recipes-kernel/kernel-tests/kernel-performance-tests.bb

```diff
@@ -11,7 +11,7 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}-files:"
 S = "${WORKDIR}"
 
 DEPENDS = "virtual/kernel"
-RDEPENDS:${PN}-ptest += "bash rt-tests fio iperf3 python3 python3-pip"
+RDEPENDS:${PN}-ptest += "bash rt-tests fio iperf3 python3 python3-pip docker"
```

Comment:
The package name might be `docker-ce` here. Could you double-check that this is correct?

Reply:
The package name is `docker-ce` but `docker-ce` provides `docker` so this works.